network-interface/src/peer_info.rs

```diff
@@ -66,9 +66,7 @@ impl Services {
                     | Services::TRANSACTION_INDEX
             }
             NodeType::Light => Services::empty(),
-            NodeType::Full => {
-                Services::ACCOUNTS_PROOF | Services::FULL_BLOCKS | Services::ACCOUNTS_CHUNKS
-            }
+            NodeType::Full => Services::ACCOUNTS_PROOF | Services::FULL_BLOCKS,
```

Comment:
Interesting. So accounts chunks can now only be received by History nodes?

Reply:
Accounts chunks can only be provided by history nodes.
They can no longer be provided by full nodes. 

This struct represents what each node type can provide

Reply:
As per our discussion, I created a new issue for this #2592